docs/core/testing/unit-testing-mstest-writing-tests-testcontext.md

```diff
@@ -0,0 +1,183 @@
+---
+title: MSTest TestContext
+description: Learn about the TestContext class of MSTest.
+author: Evangelink
+ms.author: amauryleve
+ms.date: 11/12/2024
+---
+
+# The `TestContext` class
+
+The <xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestContext> class provides contextual information and support for test execution, making it easier to retrieve information about the test run and manipulate aspects of the environment. It's defined in the <xref:Microsoft.VisualStudio.TestTools.UnitTesting> namespace and is available when using the MSTest Framework.
```

Comment:
This intro is very complicated (for me) to understand. Please try to re-word it in simpler sentences, as if the user has no context about what TestContext is, and tries to understand.

Reply:
Looking for suggestions here because that's simplest as it can get for me.

Reply:
Suggested something simpler.